storage-proofs/porep/Cargo.toml

```diff
@@ -27,10 +27,15 @@ pretty_assertions = "0.6.1"
 generic-array = "0.13.2"
 anyhow = "1.0.23"
 once_cell = "1.3.1"
-neptune = "0.5.4"
+neptune = { version = "0.5.5", features = ["gpu"] }
 num_cpus = "1.10.1"
 
 [dev-dependencies]
 tempdir = "0.3.7"
 tempfile = "3"
 rand_xorshift = "0.2.0"
+
+[features]
+default = ["column-builder-gpu"]
+#default = []
+column-builder-gpu = []
```

Comment:
given that we always build this, we might as well not add a feature

Reply:
Right, that changed -- I'll update that.